### ISSUE TITLE
requirements file incomplete

### ISSUE THREAD

**Issue by blythed** [iamalbert/pytorch-wordemb#2]
```
  Downloading torchwordemb-0.0.8.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/wm/rqr_vbc104g66ffx__hwnw3dkl2z1b/T/pip-build-3KaXlf/torchwordemb/setup.py", line 17, in <module>
        import build
      File "build.py", line 2, in <module>
        from torch.utils.ffi import create_extension
      File "/Users/dblythe/.environments/torch/lib/python2.7/site-packages/torch/utils/ffi/__init__.py", line 14, in <module>
        raise ImportError("torch.utils.ffi requires the cffi package")
    ImportError: torch.utils.ffi requires the cffi package
```
    
This is fixed by ```pip install cffi```

**Comment by MSusik** [comment]
That's actually interesting. I stumbled upon the same issue and installed `cffi` manually just like you. Then I thought it would be nice to submit a quick PR that adds `cffi` into `setup_requires`, but it turns out it's already there!

So, I digged a bit more, and it turns out that the import of the `cffi` package comes before the `setup` command. Not a trivial problem to solve, one possible solution is shown here: https://github.com/libgit2/pygit2/pull/407

As I haven't worked with `cffi` before I leave the fix to sb else.
